packages/headless/src/api/search/search/product-recommendation.ts

```diff
@@ -101,8 +101,8 @@ export interface ProductRecommendation {
 
 // Change this list when changing the fields exposed by `ProductRecommendation`
 export const ProductRecommendationDefaultFields: string[] = [
-  'uri',
-  'urihash',
+  'documentUri',
+  'documentUriHash',
```

Comment:
This is unrelated to the problem but it was bugging me.

Reply:
These fields were already returned as `documentUri` and `documentUriHash`.

I'll bring back the old ones and keep these ones then to not break anything.